src/rpc/rawtransaction.cpp

```diff
@@ -260,6 +269,69 @@ static UniValue getrawtransaction(const JSONRPCRequest& request)
     TxToJSON(*tx, hash_block, mempool, chainman.ActiveChainstate(), *llmq_ctx.clhandler, *llmq_ctx.isman, result);
     return result;
 }
+static UniValue gettransactionsarelocked(const JSONRPCRequest& request)
+{
+    RPCHelpMan{
+        "gettransactionsarelocked",
+        "\nReturn the raw transactions data.\n"
+        "\nHint: Use gettransaction for wallet transactions.\n",
+        {
+            {"txids", RPCArg::Type::ARR, RPCArg::Optional::NO, "The transaction ids (no more than 100)",
+                {
+                    {"txid", RPCArg::Type::STR_HEX, RPCArg::Optional::OMITTED, "A transaction hash"},
+                },
+            },
+        },
+        RPCResult{
+                RPCResult::Type::ARR, "", "Response is an array with the same size as the input txids",
+                {
+                        {RPCResult::Type::OBJ, "", "",
+                         {
+                                {RPCResult::Type::STR_HEX, "txid", "The transaction hash"},
+                                {RPCResult::Type::BOOL, "instantlock", "Current transaction lock state"},
+                                {RPCResult::Type::BOOL, "chainlock", "he state of the corresponding block chainlock"},
```

Comment:
Maybe smth like this...
```suggestion
                                {RPCResult::Type::BOOL, "chainlock", "Chainlock status for the block containing the transaction"},
```